### PR TITLE
Add a batch of new gallery items

### PR DESCRIPTION
**Pull request by bryevdv**
This PR adds several new examples to the gallery.

### CODE REVIEW

File: docs/bokeh/source/docs/gallery.json [hunk 1]
```diff
@@ -100,7 +108,15 @@
   "basic/annotations": [
     {
       "name": "legend.py",
-      "alt": "Thumbnail link to the examples/basic/annotations/legend.py example shows a row of two plots that have sine waves in different scatter and line styles, illustrated in accompanying legends."
+      "alt": "Thumbnail link to the examples/basic/annotations/legend.py example shows a row of fo plots that have sine waves in different scatter and line styles, illustrated in accompanying legends."
```
Comment (mosc9575) [inline docs/bokeh/source/docs/gallery.json:111]:
```suggestion
      "alt": "Thumbnail link to the examples/basic/annotations/legend.py example shows a row of two plots that have sine waves in different scatter and line styles, illustrated in accompanying legends."
```

Some typo.

File: docs/bokeh/source/docs/user_guide/basic/axes.rst [hunk 2]
```diff
@@ -108,3 +108,15 @@ adding new glyph methods as well as when adding new axis objects with the
 
 .. bokeh-plot:: __REPO__/examples/basic/axes/twin_axes.py
     :source-position: above
+
+.. _ug_basic_axes_fixed:
+
+Fixed location axis
+-------------------
+
+By defatult Bokeh places axes on the sides of plots, but it is possible to
```
Comment (mosc9575) [inline docs/bokeh/source/docs/user_guide/basic/axes.rst:117]:
```suggestion
By default Bokeh places axes on the sides of plots, but it is possible to
```

**Review by mosc9575 — COMMENTED** [review]
I found some simple typos.

File: docs/bokeh/source/docs/user_guide/basic/axes.rst (hunk 2 above)
Comment (bryevdv) [reply docs/bokeh/source/docs/user_guide/basic/axes.rst:117]: oops, I missed seeing these before merging. Planning to add more examples, I will get to them in those PRs

**Pull request merged**